tests/bfd/conftest.py

```diff
@@ -111,4 +113,32 @@ def bfd_cleanup_db(
             else request.config.dst_asic
         )
         for interface in selected_interfaces:
-            bfd_base_instance.interface_cleanup(dut, asic, interface)
+            ensure_interface_is_up(dut, asic, interface)
```

Comment:
The `bfd_base_instance.interface_cleanup()` function was absent originally. Based on the tests and also the logging message above, I assume we want to ensure all the interfaces are up after tests on portchannels or portchannel members, so I added a `ensure_interface_is_up()` function below.